spec.md

```diff
@@ -1593,6 +1593,39 @@ message GetCapacityResponse {
   // storage. This field is REQUIRED.
   // The value of this field MUST NOT be negative.
   int64 available_capacity = 1;
+
+  // The largest size that may be used in a
+  // CreateVolumeRequest.capacity_range.required_bytes field
+  // to create a volume with the same parameters as those in
+  // GetCapacityRequest.
+  //
+  // If `volume_capabilities` or `parameters` is
+  // specified in the request, the Plugin SHALL take those into
+  // consideration when calculating the maximum volume size of the
+  // storage.
+  //
+  // This field is OPTIONAL. COs MAY use it to make decision about
+  // where to create volumes.
+  Capacity maximum_volume_size = 2 [(alpha_field) = true];
+
+  // The smallest size that may be used in a
+  // CreateVolumeRequest.capacity_range.limit_bytes field
+  // to create a volume with the same parameters as those in
+  // GetCapacityRequest.
+  //
+  // If `volume_capabilities` or `parameters` is
+  // specified in the request, the Plugin SHALL take those into
+  // consideration when calculating the maximum volume size of the
```

Comment:
s/maximum/minimum/ ?

Reply:
Darn cut-and-paste. Fixed.

Reply:
It's actually correct here and wrong for `maximum_volume_size`: the requested minimum volume size must be <= `maximum_volume_size` and the requested maximum volume size (this case here, aka `capacity_range.limit_bytes`) >= `minimum_volume_size`.

I fixed the `maximum_volume_size` comment.